lib/widget/helpers/controllers.dart

```diff
@@ -27,6 +27,7 @@ abstract class WidgetController extends Controller {
   @override
   Map<String, Function> getBaseSetters() {
     return {
+      'id': (value) => id = Utils.optionalString(value),
```

Comment:
you don't need this. ID is added to the Scope's data context. It is handled outside the widget. We do need the schema to reflect the ID though

Reply:
good point, will close it